hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -789,4 +796,28 @@ public static String getOMAddressListPrintString(List<OMNodeDetails> omList) {
     printString.append("]");
     return printString.toString();
   }
+
+  public static String format(List<ServiceInfo> nodes, int port) {
```

Comment:
Instead of multiple fields like Current Role, leaderID , we can rather add this info in this output itself, just add a logic like 

```
public static String format(List<ServiceInfo> nodes,
      int port, String leaderId) {
    StringBuilder sb = new StringBuilder();
    int count = 0;
    for (ServiceInfo info : nodes) {
      // Printing only the OM's running
      String role = info.getOmRoleInfo().getNodeId().equals(leaderId)?"LEADER":"FOLLOWER";
      if (info.getNodeType() == HddsProtos.NodeType.OM) {
        sb.append(
            String.format(
                " { HostName: %s | Node-Id: %s | Ratis-Port : %d | Role : %d } ",
                info.getHostname(),
                info.getOmRoleInfo().getNodeId(),
                port,
                role
            ));
```

Reply:
Thanks, I have added the role-info to the ratis ring !!